apps/user-service/src/users/users.controller.ts

```diff
@@ -14,10 +15,13 @@ import { PolicyService } from "@terramatch-microservices/common";
 import { ApiOperation, ApiParam } from "@nestjs/swagger";
 import { OrganisationDto, UserDto } from "@terramatch-microservices/common/dto";
 import { ExceptionResponse, JsonApiResponse } from "@terramatch-microservices/common/decorators";
-import { buildJsonApi, DocumentBuilder } from "@terramatch-microservices/common/util";
+import { buildJsonApi, DocumentBuilder, JsonApiDocument } from "@terramatch-microservices/common/util";
 import { UserUpdateBodyDto } from "./dto/user-update.dto";
+import { NoBearerAuth } from "@terramatch-microservices/common/guards";
+import { UserNewRequest } from "./dto/user-new-request.dto";
+import { UserCreationService } from "./user-creation.service";
 
-const USER_RESPONSE_SHAPE = {
+export const USER_RESPONSE_SHAPE = {
```

Comment:
Does this need to be exported? I don't think it should be used outside of this file and I don't see any users external to this file in this PR.